index.js

```diff
@@ -1 +1,11 @@
-var root = document.querySelector('#root');
\ No newline at end of file
+var root = document.querySelector('#root');
+// image we have four compnets that we have from source directory 
+// use these components to overright inner html of root element 
+function render(state){
```

Comment:
This is looking good! But if you're going to create this function, you also need to _call_ this function on the last line of the file.